sopel/modules/tld.py

```diff
@@ -9,7 +9,7 @@
 """
 from __future__ import absolute_import, division, print_function, unicode_literals
 
-from datetime import datetime
+from datetime import datetime, timezone
```

Comment:
```suggestion
from datetime import datetime
```

Reply:
HA I can't do the other suggestion, which is to do that:

```
import pytz
```

Right before the `import requests` down below.

Reply:
Gotta love GitHub's artificial restrictions on PR reviews. 🙄